packages/hebao_v1/contracts/modules/security/WhitelistModule.sol

```diff
@@ -0,0 +1,113 @@
+/*
+
+  Copyright 2017 Loopring Project Ltd (Loopring Foundation).
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+pragma solidity ^0.5.11;
+
+import "../../iface/Wallet.sol";
+
+import "../stores/WhitelistStore.sol";
+
+import "./SecurityModule.sol";
+
+
+/// @title WhitelistModule
+/// @dev Manages whitelisted addresses.
+contract WhitelistModule is SecurityModule
+{
+    WhitelistStore  public whitelistStore;
+    SecurityStore   public securityStore;
```

Comment:
`securityStore` is already defined in SecurityModule. Having a variable here with the same name is actually a new variable, unrelated to the one in the super class. This means `securityStore` is undefined in WhitelistModule. To make `securityStore` public in WhitelistModule a getter function needs to be used .

```
contract SuperContract
{
    uint internal a;
    
    constructor(uint _a) public {
        a = _a;
    }
    
    function getA() external view returns (uint) {
        return a;
    }
}

contract BaseContract is SuperContract
{
    uint public a;
    
    constructor() public 
        SuperContract(20)
    {
        a = 10;
    }
}
```

`BaseContract.a() == 10`
`BaseContract.getA() == 20`